filters/auth/authclient.go

```diff
@@ -36,9 +38,9 @@ type tokeninfoClient interface {
 
 var _ tokeninfoClient = &authClient{}
 
-func newAuthClient(baseURL, spanName string, timeout time.Duration, maxIdleConns int, tracer opentracing.Tracer) (*authClient, error) {
+func newAuthClient(baseURL, spanName string, timeout time.Duration, maxIdleConns int, tracer trace.Tracer) (*authClient, error) {
 	if tracer == nil {
-		tracer = opentracing.NoopTracer{}
+		tracer = &tracing.TracerWrapper{Ot: opentracing.NoopTracer{}}
```

Comment:
why not https://pkg.go.dev/go.opentelemetry.io/otel/trace@v1.24.0/noop#Tracer ?

Reply:
Yeah, I replaced this in a few places, I must have missed this one. Thanks :)